styleguide/js/zoolander.js

```diff
@@ -21,14 +21,41 @@ jQuery(document).ready(function ($) {
     jQuery ('.title a').click(function (event) {
       jQuery (this).parent().parent().find('.fa-arrow-right').animate({  borderSpacing: +90 }, {
       step: function(now,fx) {
-          $(this).css('-webkit-transform','rotate('+now+'deg)'); 
+          $(this).css('-webkit-transform','rotate('+now+'deg)');
           $(this).css('-moz-transform','rotate('+now+'deg)');
           $(this).css('transform','rotate('+now+'deg)');
           counter = 1;
       },
       duration:'slow'
       },'linear');
     });
-});
-
 
+    mobileFooterCheck();
+    $(window).resize(function(){
+      mobileFooterCheck();
+    });
+    function mobileFooterCheck(){
+      var footerSectionId = 0;
+      $('.footer ul li span.menu-title').each(function(){
+        footerSectionId++;
+        var _this = $(this);
```

Comment:
It's a common practice to name a variable referencing a jQuery wrapped object with a $, because it helps the person reading the code know you are dealing with jQuery and not a common JavaScript variable. 

I'd recommend `var $this = $(this);` instead of `var _this = $(this);`

See: http://stackoverflow.com/questions/6209462/when-why-to-prefix-variables-with-when-using-jquery